kubelet/tests/test_kubelet.py

```diff
@@ -449,24 +455,24 @@ def test_report_container_spec_metrics(monkeypatch, tagger):
         mock.call(
             'kubernetes.cpu.requests',
             0.1,
-            instance_tags + ['kube_container_name:fluentd-gcp', 'kube_deployment:fluentd-gcp-v2.0.10'],
+            ['kube_container_name:fluentd-gcp', 'kube_deployment:fluentd-gcp-v2.0.10'] + instance_tags,
```

Comment:
what's the reason behind these changes?

Reply:
With the change, the instance tags are added at the end of the tags, not at the beginning.